_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,419 @@
+---
+id: be4d0c
+name: Inline link has distinguishable style not based on color (hue) alone
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style not based on color (hue) alone.
+accessibility_requirements:
+  wcag20:1.4.1: # Use of Color (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+  previous_authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to each [link history state][] of any [semantic link][], for which all the following is true:
```

Comment:
Also, slight consequence of having the link history state in Applicability: each test case now has two targets ((link, `:link`) and (link, `:visited`)) but the descriptions do not really reflect that. We can probably cover this globally since it is the case for each case (in https://act-rules.github.io/rules/oj04fd#background and #421, I did it with a note at the end of the Background section, i.e. just before the test cases, but other ways are possible).

This does ripple a bit to Failed Example 2 where one of the targets (the `:link` one) passes the rule, but the other fails it.

Reply:
Hold on a minute, reading your comment I've come in doubt about this "link history state" thing 🤔 What does it mean for semantic link to have a `:link` or `:visited` state? I initially read it as "when the semantic link matches `:link` or `:visited`", but that will not work as those selectors will only ever match actual hyperlinks, such as `<a>` and `<link>` elements, not something like `<span role="link">`.

Reply:
Good catch @kasperisager 
I've removed the "link history state" from the applicability and created a new expectation to deal with it when the semantic link can have the `:link`and `:visited` pseudo-classes.
This also solved the issue @Jym77 was raising, since the "list history state" was removed from the applicability.